.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: Test
+
+on: push
+
+env:
+  COMPOSE_FILE: docker-compose.test.yml
+  DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+  DOCKER_IMAGE: ${{ github.repository }}
+  DOCKER_REGISTRY: ghcr.io
+  DOCKER_USERNAME: ${{ github.repository_owner }}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Branch Tag Action
+        uses: nimblehq/branch-tag-action@v1.1
+
+      - name: Docker login to Github package registry
```

Comment:
A small thing, is it `Github package registry` 🤔

Reply:
It is Github container registry. My bad 😆

Reply:
Fixed